moped-editor/src/views/projects/projectsListView/ProjectsListViewTable.js

```diff
@@ -336,6 +361,7 @@ const ProjectsListViewTable = ({ title, query, searchTerm, referenceData }) => {
       title: "Signal IDs",
       field: "project_feature",
       hidden: hiddenColumns["project_feature"],
+      sorting: false,
```

Comment:
Thanks for disabling the ordering here. It makes sense to me that because we have one-or-more signal ids that can come back in this column, ordering isn't really a sensical operation. 👍